homeassistant/components/google_assistant/trait.py

```diff
@@ -1397,6 +1413,30 @@ def sync_attributes(self):
                 }
             )
 
+            if self._ordered_speed:
+                result.update(
+                    {
+                        "availableFanSpeeds": {
+                            "speeds": [
+                                {
+                                    "speed_name": speed,
+                                    "speed_values": [
+                                        {
+                                            "speed_synonym": [
+                                                f"Speed {speed}",
```

Comment:
Should we also include low/medium/high?

Reply:
We could create fixed sets for 2, 3, 4, 5 speed sets. But i had a hard time figuring out good names for 4 and 5 sets.

Reply:
Ps. Its limited to 5 steps at the moment. 4 would likely be an okey limit too.

Reply:
I added names to the speeds. I agree it is better. I wonder if we could move the synonyms to strings.json, and add an option to select which translations to provide to google.

Reply:
let's stick with this. We can do that once we have backend translations figured out (we don't have a language setting at core level right now, just user)